terraform/modules/nitro/main.tf

```
@@ -0,0 +1,38 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_instance" "nitro" {
+  ami               = "ami-0dfcb1ef8550277af"
+  instance_type     = "c5a.xlarge"
+  key_name          = var.key_pair_name
+  security_groups   = [aws_security_group.nitro.name]
+
+
+  tags = {
+    Name = var.instance_name
+  }
+
+  enclave_options {
+    enabled = true
+  }
+}
+
+resource "aws_security_group" "nitro" {
+  name = "${var.instance_name}-security-group"
+
+  ingress {
```

I've found that adding a description indicating the purpose/intention of security rules helps a lot for later readers
```suggestion
  ingress {
    description = "SSH from anywhere"
```